README.md

```diff
@@ -17,21 +17,46 @@ A super simple Java project that utilizes DALL-E3, LangChain4J, and Azure Open A
     ```bash
     git clone https://github.com/sandraahlgrimm/langchain4j-dalle3-demo.git
     ```
+    
+    These instructions were tested to work with the code as of tag `20240508`. The instructions should work with the `HEAD` commit, but if you run into problems, try checking out the tag.
 
-2. Create an Azure Open AI account and get the required credentials:
+2. Create an Azure Open AI account and get the required credentials.
+
+   1. In a new tab, visit [Create and deploy an Azure OpenAI Service resource](https://learn.microsoft.com/en-us/azure/ai-services/openai/how-to/create-resource?pivots=web-portal).
+   1. If you want to use the following steps, select the **Portal** tab. Otherwise, just follow the documentation to get the following environment variable values in the way best suited to your needs.
+
+      - `AZURE_OPENAI_ENDPOINT`: Your Azure OpenAI Endpoint.
+      - `AZURE_OPENAI_DEPLOYMENT_NAME`: Your Azure Open AI Deployment name.
+      - `AZURE_OPENAI_API_KEY`: Your Azure Open AI API key.
+      
+   1. Follow the steps up to and including the section **Deploy a model**.
+   
+   1. When you get to the step asking you to **Create new deployment**, use the following substitutions.
+   
+      1. For **Select a model** select **dall-e-3**.
+      1. For **Deployment name** use the same value as your resource group name.
 
     - [Azure Open AI Documentation](https://learn.microsoft.com/azure/ai-services/openai/how-to/create-resource?pivots=cli)
+
+    1. Ensure the checkmark next to your deployment is selected and select **Open in playground**.
+    1. Select **View code**. Keep this screen open and proceed.
  
-3. Set up the environment variables in an `.env` file in the root directory of the project with the following content:
+3. Set up the environment variables in an `.env` file in the root directory of the project with the following content. Adjust the syntax for your operating system if it is not [POSIX compliant](https://posix.opengroup.org/).
 
-    - `AZURE_OPENAI_ENDPOINT`: Your Azure OpenAI Endpoint.
-    - `AZURE_OPENAI_DEPLOYMENT_NAME`: Your Azure Open AI Deployment name.
-    - `AZURE_OPENAI_API_KEY`: Your Azure Open AI API key.
+```bash
+export AZURE_OPENAI_ENDPOINT=<select the copy icon on the Endpoint in Azure AI Studio playground>
+export AZURE_OPENAI_DEPLOYMENT_NAME=<your resource group name>
```

Comment:
It's the Azure OpenAI Deployment name. Not the resource group.

Reply:
But why invent a new string? I am suggesting to use the resource group name as the deployment name. There is no technical reason why they cannot be the same and it's one less thing to keep track of. Of course, this is just my suggestion. Feel free to decline it.

Reply:
Once we have more than one deployment (like for more than one model) we need to use a new one. Also, model name and deployment name to avoid conflicts we have seen with frameworks like SpringAI